### ISSUE TITLE
Cannot connect mysql from local

### ISSUE THREAD

**Issue by cxq914** [docker-library/mysql#846]
When i installed mysql 8.0-oracle from dockerhub: I cannot connect mysql server from local:
<img width="830" alt="image" src="https://user-images.githubusercontent.com/7677708/164792640-f70c6897-145e-4303-aa8b-bd024e6ddc86.png">

Things have been tried:

- update bind address in my.conf: ended with the issue that:

- all linux commands not found: sudo, yum, vi, which...only few works: cat, find, ls... here is my linux distribution
<img width="341" alt="image" src="https://user-images.githubusercontent.com/7677708/164792988-bb1947c8-7b12-472f-a784-8f39c3a2c9b9.png">

**Comment by wglambert** [comment]
Can you give all the commands you ran and any relevant files or logs for reproducing the issue

You could also try asking over at the [Docker Community Forums](https://forums.docker.com/), [Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker). Since these repos aren't really a user-help forum

**Comment by cxq914** [comment]
Hi @wglambert  Thank you so much for your comments. The links you provided is really helpful. Just joined Docker Community for more resources.

BTW, my problem was solved. I uninstalled my previous mysql and re-installed mysql via the code below:

`docker run --name **-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=*** -d mysql:8.0.28-oracle`

- In this case, i added the port information specifically: `-p 3306:3306`. which solved my connection failure.
